sections/introduction.include

```diff
@@ -1149,19 +1149,27 @@
       confusing combinations are disallowed.
 
       <p class="example">
-        For example, <{div}> elements are rendered as <a>block boxes</a>, and <{span}> elements as
-        <a>inline boxes</a>. Putting a <a>block box</a> in an inline box is unnecessarily confusing;
-        since either nesting just <{div}> elements, or nesting just <{span}> elements, or nesting
-        <{span}> elements inside <{div}> elements all serve the same purpose as nesting a <{div}>
-        element in a <{span}> element, but only the latter involves a <a>block box</a> in an
-        <a>inline box</a>, the latter combination is disallowed.
+        For example, <{div}> elements are rendered as <a>block boxes</a>, and <{span}> elements
+        as <a>inline boxes</a>. Putting a <a>block box</a> in an inline box is unnecessarily
+        confusing; since either nesting just <{div}> elements, or nesting just <{span}> elements,
+        or nesting <{span}> elements inside <{div}> elements all serve the same purpose as nesting
+        a <{div}> element in a <{span}> element, but only the latter involves a <a>block box</a>
+        in an <a>inline box</a>, the latter combination is disallowed.
       </p>
 
       <p class="example">
-        Another example would be the way <a>interactive content</a> cannot be nested. For example, a
-        <{button}> element cannot contain a <{textarea}> element. This is because the default
-        behavior of such nesting interactive elements would be highly confusing to users. Instead of
-        nesting these elements, they can be placed side by side.
+        Another example would be the way <a>interactive content</a> cannot be nested. For example,
+        a <{button}> element cannot contain a descendant <{textarea}> element. This is because the
+        default behavior of nesting interactive elements would be highly confusing to users.
+        Rather than nesting these elements, they could instead be styled with CSS to be visually
+        placed side by side.
+      </p>
+
+      <p class="note">
+        An exception to disallowing <a>block box</a> within <a>inline box</a> rendered elements
```

Comment:
Is this the only exception, or are there others (which is what seems to be implied here)?